tools/codegen/gen.py

```diff
@@ -481,6 +482,8 @@ def gen_unstructured(self, f: NativeFunction) -> Optional[str]:
 
         if self.dispatch_key not in f.dispatch:
             return None
+        if f.manual_kernel_registration:
```

Comment:
Is this change actually changing what gets generated or not? From what I can see, it is not equivalent to the removal from `ComputeFunction` below right?

Reply:
Yes, I am generating less code. Fortunately the code was all dead so there is no visible impact.